lib/TabTracker.js

```diff
@@ -0,0 +1,89 @@
+const tabs = require("sdk/tabs");
+const ss = require("sdk/simple-storage");
+
+const NEWTAB_URL = "resource://activity-streams/data/content/activity-streams.html#/";
```

Comment:
Actually, let me refine my comment:

we have 2 urls we care about, the "activity streams" page and the "newtab" page. We want to track engagement on both separately.

@k88hudson can advise how the URLs differ